src/util/imagefiledata.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <QByteArray>
+#include <QImage>
+
+class ImageFileData : public QImage {
+  public:
+    explicit ImageFileData(const QByteArray& coverArtBytes,
+            const QByteArray& coverArtFormat = nullptr);
+
+    static ImageFileData fromFilePath(const QString& coverArtFilePath);
+
+    static QByteArray getFormat(const QByteArray& coverArtBytes);
```

Comment:
The function name getXXX is "reserved" for getter functions, like 
```
QByteArray getFormat() const {
   return m_coverArtFormat;
```
Internally it defaults to someting like
```
QMimeDatabase().mimeTypeForFile(
            fileInfo, QMimeDatabase::MatchContent); 
```
how about rename this to `readFormat(..)` or `formatFromContent(..)` or `getFormatFrom(..)`

Reply:
Thank you for the information.

changed to readFormatFrom(..) 👍